waku/v2/waku_discv5.nim

```diff
@@ -77,9 +95,18 @@ proc new*(T: type WakuDiscoveryV5,
           flags: CapabilitiesBitfield,
           multiaddrs = newSeq[MultiAddress](),
           rng: ref HmacDrbgContext,
-          discv5Config: protocol.DiscoveryConfig = protocol.defaultDiscoveryConfig): T {.
+          topics: seq[string],
+          discv5Config: protocol.DiscoveryConfig = protocol.defaultDiscoveryConfig
+          ): T {.
   deprecated: "use the config and record proc variant instead".}=
 
+  let relayShardsRes = topicsToRelayShards(topics)
+
+  let relayShard: Option[RelayShards] = if relayShardsRes.isErr():
+    debug "pubsub topic parsing error", reason = relayShardsRes.error
+    none(RelayShards)
+  else: relayShardsRes.get()
```

Comment:
Nitpick:

Perhaps possible to indent the `if` and `else` to be under each other and indented from the `let`.
The following is just a suggestion but may be one way to do it:

```suggestion
  let relayShard =
    if relayShardsRes.isErr():
      debug "pubsub topic parsing error", reason = relayShardsRes.error
      none(RelayShards)
    else: relayShardsRes.get()
```

Reply:
I agree, your version is more readable.